plugins/modules/vmware_dvs_portgroup.py

```diff
@@ -17,205 +18,231 @@
     - Create or remove a Distributed vSwitch portgroup.
 author:
     - Joseph Callen (@jcpowermac)
-    - Philippe Dellaert (@pdellaert) <philippe@dellaert.org>
+    - Philippe Dellaert (@pdellaert)
+    - Christian Kotte (@ckotte)
 notes:
     - Tested on vSphere 5.5, 6.5
 requirements:
     - "python >= 2.6"
     - PyVmomi
 options:
-    portgroup_name:
+    portgroup:
         description:
             - The name of the portgroup that is to be created or deleted.
         required: True
         type: str
-    switch_name:
+        aliases: ['portgroup_name']
+    switch:
         description:
             - The name of the distributed vSwitch the port group should be created on.
         required: True
         type: str
-    vlan_id:
+        aliases: ['switch_name']
+    description:
         description:
-            - The VLAN ID that should be configured with the portgroup, use 0 for no VLAN.
-            - 'If C(vlan_trunk) is configured to be I(true), this can be a combination of multiple ranges and numbers, example: 1-200, 205, 400-4094.'
-            - The valid C(vlan_id) range is from 0 to 4094. Overlapping ranges are allowed.
-            - 'If C(vlan_private) is configured to be I(true), the corresponding private VLAN should already be configured in the distributed vSwitch.'
-        required: True
+            - The description of the portgroup.
         type: str
+        aliases: ['portgroup_description']
     num_ports:
         description:
             - The number of ports the portgroup should contain.
-        required: True
+            - The number of ports for a new portgroup will be 0 if not specified.
+            - The vSphere Client uses 8 by default.
+            - This option will be ignored if C(port_binding) is set to 'ephemeral'.
```

Comment:
```suggestion
            - This option will be ignored if I(port_binding) is set to C(ephemeral).
```

Reply:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation